tests/test_db_function.py

```diff
@@ -228,26 +228,161 @@ def test_update_workflow_status_on_insert(
 
 
 @pytest.mark.apps_to_insert([test_input_data[0]])
-def test_bulk_update_location_data(_db, seed_application_records):
+def test_bulk_update_location_data_with_valid_value(
```

Comment:
Instead of 3 test cases, can we parametrize the test `test_bulk_update_location_data` with 3 testdata sets ?

Reply:
Thanks, @RamuniN 
I have parametrized tests